.github/workflows/test-python.yml

```diff
@@ -30,14 +30,14 @@ jobs:
           mongodb-replica-set: test-rs
       - name: Install libbson
         run: |
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson ./build-libbson.sh
```

Comment:
Why rename LIBBSON_INSTALL_DIR to MONGO_LIBBSON_DIR? Asking because it makes this PR a little more difficult to review.

Reply:
Only for consistency with the arrow directory, I'll revert.